src/test/java/com/cloudant/tests/ReplicatorTest.java

```diff
@@ -138,15 +135,20 @@ public void replication_conflict() throws Exception {
         foodb1.setTitle("titleZ");
         db1.update(foodb1);
 
-        account.replicator().source(db1URI)
+        Response secondResponse = account.replicator().source(db1URI)
                 .target(db2URI).save();
 
-        Thread.sleep(1000);
+        // we need the replication to finish before continuing
+        Utils.waitForReplicatorToComplete(account, response);

```

Comment:
Doesn't this one need to be `waitForReplicatorToComplete(account, secondResponse)`?